modules/ROOT/pages/how-to-export-resources.adoc

```diff
@@ -26,10 +28,16 @@ To export Java classes, use the `exportedPackages` field of the `mule-artifact.j
 }
 ----
 
-CAUTION: Export the minimum number of classes required for your application to work. This will reduce the likelihood of conflicts with other artifacts, such as modules or connectors, that you may be using within your application.
+Any internal dependencies in your Java classes are automatically exported with this configuration.
 
-TIP: Try to use `.api.` within public packages and `.internal.` within your non-public packages.
+=== Considerations
 
+* Export the minimum number of classes required for your application to work
```

Comment:
Given that only packages can be exported, how can a user ensure that they are exportign the "minimum number of classes"?